services/distribution/src/test/java/app/coronawarn/server/services/distribution/assembly/component/BoosterNotificationStructureProviderTest.java

```diff
@@ -56,13 +58,14 @@
 
   @Test
   void shouldCreateCorrectFileStructureForBoosterNotificationBusinessRules() {
-    Writable<WritableOnDisk> boosterNotificationRules = underTest.getBoosterNotificationRules();
-    boosterNotificationRules.prepare(new ImmutableStack<>());
+    Optional<Writable<WritableOnDisk>> boosterNotificationRules = underTest.getBoosterNotificationRules();
```

Comment:
Sorry for being picky. Would it be possible to have a given-when-then structure? So moving the assertions to the bottom. It would help gaining a top level view of the tests